spread.meta.yaml

```diff
@@ -0,0 +1,29 @@
+# This file is used to define which features, groups and tags are available in spread tests
+# All these definitions are cross-checked to ensure tests integrity.
+# These definitions can be used to define a subset of tests to run but also as a way
+# of coverage documentation.
+
+definitions:
+    # Features refer to product caracteristics being tested
+    # The main porpoise is to know the features testing coverage
+    features: [
+        interfaces,           # snap interfaces
+        snap_refresh,         # snap refresh and snap auto refreshes
+        snap_install          # snap install command
+        ]
+
+    # Groups are set of tests orthogonal to the suites definition
```

Comment:
Perhaps we can explain "orthogonal" in more practical terms e.g. set of tests that can be selected across multiple suites? 

Assuming that's what is meant, is this not true to tags and features as well?